src/styles/cabincup.scss

```diff
@@ -0,0 +1,28 @@
+
+
+h2{
```

Comment:
Would you mind wrapping putting all of these class definitions inside of the .cabin-cup definition - just so they don't accidentally get applied to everything on the page? You could do something like:

```
.cabin-cup {
    background-color: #1b214e;
    border-radius: 15px;

    h2 {
        color: white;
        font-weight: bold;
        padding-top: 20px;
    }

    h5 {
        color: white;
    }
    ...
}
```
And that will ensure that they only get applied to this section.